jekyll/_cci2/using-dynamic-configuration.adoc

```diff
@@ -196,22 +198,432 @@ workflows:
+
+[#code-config-1]
+==== `code-config.yml`
+
+[,yaml]
+----
+version: 2.1
+
+jobs:
+  build-code:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - run: echo "Building code"
+
+workflows:
+  code-workflow:
+    jobs:
+      - lint # use the shared lint job.
+      - test # use the shared test job.
+      - build-code
+----
+
+[#docs-config-1]
+==== `docs-config.yml`
+
+[,yaml]
+----
+version: 2.1
+
+jobs:
+  build-docs:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - run: echo "Building docs"
+
+workflows:
+  docs-workflow:
+    jobs:
+      - lint # use the shared lint job.
+      - build-docs
+----
+
+[#config-no-change-1]
+=== Configuration for no change
+
+We have used the path filtering orb to map configuration files with file paths so we know which configuration will be used when a change is made in a specific location. If a pipeline is triggered with no changes in any of our defined paths, CircleCI needs to know what to do next.
+
+One way to handle this scenario is to provide an alternative configuration using the `config-path` parameter (see line 27 <<setup-config-1,here>>). `config-path` is ignored if a mapping is in place, but will be used if no mapping matches the path to modified files. In this example, we use a `no-updates.yml` configuration file. Another option would be to configure a step using the `finish` command from the link:https://circleci.com/developer/orbs/orb/circleci/continuation#commands-finish[continuation orb].
+
+[,yml]
+----
+version: 2.1
+
+# define the parameters from the setup config.
+parameters:
+  always-continue:
+    type: boolean
+    default: false
+  build-code:
+    type: boolean
+    default: false
+  build-docs:
+    type: boolean
+    default: false
+
+jobs:
+  no-updates:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - run: echo "No updates have been made"
+
+workflows:
+  no-update-workflow:
+    jobs:
+      - no-updates
+----
+
+[#pack-generate-validate]
+== Pack, generate, and validate a configuration file for pipeline continuation
+
+For this example, consider a project that includes:
+
+* Separate directories for code (`src/`) and docs (`doc/`).
+* A setup configuration, `config.yml`.
+* Separate configuration files for building the code and the docs, `code-config.yml` and `docs-config.yml`.
+* A `shared/` configuration directory containing our shared jobs in separate YAML files. This is used to generate `shared-config.yml` using the xref:how-to-use-the-circleci-local-cli#packing-a-config[CircleCI CLI command] `circleci config pack`
+* A `no-updates.yml` config file to be used in the event that there are no changes.
+
+[,shell]
+----
+.
+├── .circleci
+│   ├── code-config.yml
+│   ├── config.yml
+│   ├── docs-config.yml
+│   ├── no-updates.yml
+│   ├── shared
+│   │   └── jobs
+│   │       ├── lint.yml
+│   │       └── test.yml
+│   └── shared-config.yml
```

Comment:
The `shared-config.yml` will be generated with the command, this can be removed.